packages/wmr/src/lib/acorn-traverse.js

```diff
@@ -366,6 +367,10 @@ class Path {
 		this.ctx.out.appendRight(this.end, str);
 	}
 
+	insertAfter(node) {
+		// TODO: implement
```

Comment:
Will have to do some studying on how to implement this one specifically, I guess this is kind of like an appentAfter this.node?